care/emr/migrations/0036_remove_servicerequest_category_and_more.py

```
@@ -0,0 +1,28 @@
+# Generated by Django 5.1.4 on 2025-04-23 17:17
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emr', '0035_healthcareservice'),
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)